app/workers/image_callback_worker.rb

```diff
@@ -5,32 +5,19 @@ class ImageCallbackWorker
 
   class UnknownImageTypeError < StandardError; end
 
-  shoryuken_options queue: "#{ENV['RAILS_ENV']}_cms_image_callback"
+  shoryuken_options queue: Image::CALLBACK_QUEUE_NAME
 
+  # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
   def perform(_sqs_msg, job)
-    image = find_image(job['type'], job['id'])
-    image.filename = job['name']
-    image.size = job['size']
-    image.width = job['width']
-    image.height = job['height']
-    image.aspect_ratio = image.width / image.height.to_f if image.width && image.height
-
-    mime_type = MIME::Types.type_for(job['format'] || '').first.try(:content_type)
-    image.content_type = mime_type || job['format']
-
-    if !job['downloaded']
-      image.status = NOTFOUND
-    elsif !job['valid']
-      image.status = INVALID
-    elsif !job['resized']
-      image.status = FAILED
-    else
-      image.status = COMPLETE
+    if job['JobResult']
+      perform_porter_job_callback(job)
+    elsif job['type'] && job['id']
```

Comment:
This still supports the old cms-image-lambda callbacks?  Does it need to?  I guess there could be some straggling messages in there during deploy or something.